bin/trin/src/lib.rs

```diff
@@ -146,7 +148,7 @@ pub async fn run_trin(
             &discovery,
             utp_socket.clone(),
             portalnet_config.clone(),
-            storage_config_factory.create(&Subnetwork::History)?,
+            storage_config_factory.create(&Subnetwork::History, trin_config.max_radius)?,
```

Comment:
Is there any reason to leave this as `Distance::MAX`? Since it seems like the bottlenecking volume is only on the state network iiuc.

Reply:
The bottlenecking is only more extreme on State. This issue occurs on History too, it is just the radius's are settled now, if we are targeting low hardware requirements I think this setting should be used on the history network as well.

Reply:
Also since we run History and State at the same time, they affect each other in terms of performance, so we should do both as bad performance on the History network would degrade performance of the State network.